src/System.CommandLine.Tests/ParserTests.MultipleArguments.cs

```diff
@@ -176,6 +176,36 @@ public void arity_ambiguities_can_be_differentiated_by_type_convertibility()
                       .BeEquivalentTo(new[] { "one", "two" },
                                       options => options.WithStrictOrdering());
             }
```

Comment:
These tests probably belong under `OptionTests`. (Per the pedantic distinction from earlier, the new functionality is about multiple tokens for a single option argument. Options don't support multiple `Argument` instances, which is what this test class is otherwise referring to.)